plugins/modules/cloud/docker/docker_stack_info.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018 Dario Zanzico (git@dariozanzico.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'metadata_version': '1.1'
+}
+
+DOCUMENTATION = '''
+---
+module: docker_stack_info
+author: "Jose Angel Munoz (@imjoseangel)"
+short_description: docker stack module info
+description:
+  - Manage docker stacks using the 'docker stack' command
+    on the target node (see examples).
+version_added: "2.11"
+options:
+  name:
+    description:
+      - Stack name
+    type: str
+  action:
+    description:
+      - If set to C(tasks) displays the list of stacks using the C(docker stack ls) command.
+        If not set, the module use "list" by default.
+    type: str
+    choices: ["list", "tasks"]
+    default: list
+'''
+
+RETURN = '''
+msg:
```

Comment:
The result should definitely **not** be called `msg`. `msg` is reserved for string results, not for arbitrary data structures.